Kicked build which had failed with
```
FAIL: test_transfer_first_doc_order (corehq.apps.commtrack.tests.test_xml.CommTrackBalanceTransferTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 232, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 205, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 220, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 201, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/tests/test_xml.py", line 399, in test_transfer_first_doc_order
    self.check_product_stock(self.sp, product, final, 0)
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/tests/test_xml.py", line 228, in check_product_stock
    self.assertEqual(expected_soh, latest_trans.stock_on_hand)
AssertionError: Decimal('50.0') != 150
-------------------- >> begin captured stdout << ---------------------
<function test_transfer_first_doc_order at 0x7fe8ba0196e0> failed with the following settings:
settings.TESTS_SHOULD_USE_SQL_BACKEND = True
```

--- a/corehq/apps/app_manager/views/download.py
+++ b/corehq/apps/app_manager/views/download.py
@@ -236,7 +236,7 @@ def download_file(request, domain, app_id, path):
                 if build_profile in request.app.build_profiles and build_profile_access:
                     try:
                         # look for file guaranteed to exist if profile is created
-                        request.app.fetch_attachment('files/{id}/profile.xml')
+                        request.app.fetch_attachment('files/{id}/profile.xml'.format(id=build_profile))
                     except ResourceNotFound:
                         request.app.create_build_files(save=True, build_profile_id=build_profile)
                         request.app.save()

